pytorch_forecasting/models/nbeats/_nbeats.py

```diff
@@ -30,6 +30,19 @@ def __init__(
         expansion_coefficient_lengths: Optional[List[int]] = None,
         prediction_length: int = 1,
         context_length: int = 1,
+        use_kan: bool = False,
+        num_grids: int = 5,
+        k: int = 3,
+        noise_scale: float = 0.5,
+        scale_base_mu: float = 0.0,
+        scale_base_sigma: float = 1.0,
+        scale_sp: float = 1.0,
+        base_fun: callable = torch.nn.SiLU(),
```

Comment:
Please avoid all mutable objects as defaults. Since changing these mutable objects might lead to side-effects that are difficult to debug.